toolkit/src/components/cut/list-item/service/index.hbs

```diff
@@ -61,6 +61,17 @@
               @text='permissive mTLS'
               data-test-permissive-mtls
             />
+          {{else if
+            (and
+              (eq @service.metadata.isPermissiveMTls false)
+              (not (eq @service.metadata.isPermissiveMTls undefined))
+            )
+          }}
+            <Cut::TextWithIcon
+              @icon='lock'
+              @text='strict mTLS'
+              data-test-strict-mtls
```

Comment:
Can we potentially avoid the three state boolean here?
Maybe add an assertion that `@service.metadata.isPermissiveMTls` can't be undefined?

Or maybe set it default to be true or false, and if isn't passed in, then we just assume it's one of those?
Unless this is actually three separate states we want? `strict`, `permissive` and `none of the above`? Maybe we could use an enum for that?

Reply:
Ya good callout. Originally there we thought their were only two states but turns out their is an unspecified state as well. I will update this to be an enum but I'll do it in a new PR.